### PR TITLE
Adding Deep.BI analytics to AMP

### PR DESCRIPTION
**Pull request by pkalinowski**
Follow up of https://github.com/ampproject/amphtml/pull/25128
(fixing CLA - "cannot verify e-mail" issue)

Intent-to-implement ticket: #25070
@micajuine-ho @zhouyx - Thank you for your input. I hope this time it will get smoother :)

### CODE REVIEW

**Comment by calebcordry** [comment]
Adding @ mentions as reviewers.

File: extensions/amp-analytics/0.1/vendors.js [hunk 1]
```diff
@@ -110,6 +111,8 @@ const DEFAULT_CONFIG = jsonLiteral({
     'ampdocHostname': 'AMPDOC_HOSTNAME',
     'ampdocUrl': 'AMPDOC_URL',
     'ampGeo': 'AMP_GEO',
+    'ampUserLocation': 'AMP_USER_LOCATION',
+    'ampUserLocationPoll': 'AMP_USER_LOCATION_POLL',
```
Comment (micajuine-ho) [inline extensions/amp-analytics/0.1/vendors.js]: Why are these getting added?

Comment (pkalinowski) [reply extensions/amp-analytics/0.1/vendors.js]: I must have taken an older version of the file.
Fixed! Thanks!

**Comment by pkalinowski** [comment]
Thank you very much @micajuine-ho!!!

Hello @zhouyx, I hope we'll be going with our analytics for AMP soon now :)

**Comment by pkalinowski** [comment]
Hello @calebcordry, @micajuine-ho, @zhouyx ,

As far as I can see @zhouyx is on her vacation or otherwise is not available at the moment. Could you please @calebcordry and @micajuine-ho revise the review request to @zhouyx? 
I will be the most grateful for any help in pushing my little contribution forwad :]
Thank you!

**Comment by zhouyx** [comment]
PR merged. Thanks

**Pull request merged**